locales/en/events.json

```diff
@@ -0,0 +1,412 @@
+  },
+  "PersonMoveBookedIntoReceivingEstablishment": {
+    "heading": "{{person.fullname}} booked into {{location.title}}",
+    "description": ""
+  },
+  "PersonMoveMajorIncidentOther": {
+    "heading": "Major incident occurred",
+    "description": "$t(events::incident_level_details, {\"incident_level\": \"Major\"})"
+  },
+  "PersonMoveMinorIncidentOther": {
+    "heading": "Minor incident occurred",
+    "description": "$t(events::incident_level_details, {\"incident_level\": \"Minor\"})"
+  },
+  "PersonMovePersonEscapedKpi": {
+    "heading": "KPI Escape occured",
+    "description": "$t(events::incident_escaped)"
+  },
+  "PersonMovePersonEscaped": {
+    "heading": "Non KPI Escape occured",
+    "description": "$t(events::incident_escaped)"
+  },
+  "PersonMoveReleasedError": {
+    "heading": "Released in error",
+    "description": "{{person.fullname}} was released in error from {{location.title}} $t(events::incident_reported_classified)"
+  },
+  "PersonMoveRoadTrafficAccident": {
+    "heading": "Road traffic accident",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} met with road traffic accident $t(events::incident_reported_classified)"
+  },
+  "PersonMoveSeriousInjury": {
+    "heading": "Serious injury sustained",
+    "description": "{{person.fullname}} sustained serious injury $t(events::select_incident_location_preposition) $t(events::select_incident_location_vehicle) $t(events::incident_reported_classified)"
+  },
+  "PersonMoveUsedForce": {
+    "heading": "Supplier used force",
+    "description": "{{move.supplier.name}} staff {{supplier_personnel_numbers}} used force on {{person.fullname}} <br>The incident was reported at {{reported_at, formatTime}} <br>$t(events::select_fault_classification)"
+  },
+  "PersonMoveVehicleBrokeDown": {
+    "heading": "Supplier vehicle breakdown",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} broke down $t(events::incident_reported_classified)"
+  },
+  "PersonMoveVehicleSystemsFailed": {
+    "heading": "Supplier vehicle systems failed",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} has had a system failure $t(events::incident_reported_classified)"
+  },
+  "StatusMoveBooked": {
+    "statusChange": true,
+    "heading": "$t(statuses::booked)",
+    "description": "Move has been booked with {{move.supplier.name}}"
+  },
+  "StatusMoveCancelled": {
+    "statusChange": true,
+    "heading": "$t(statuses::cancelled)",
+    "description": "Move has been cancelled"
+  },
+  "StatusMoveCompleted": {
+    "statusChange": true,
+    "heading": "$t(statuses::completed)",
+    "description": "Move has been completed by {{move.supplier.name}}"
+  },
+  "StatusMoveInTransit": {
+    "statusChange": true,
+    "heading": "$t(statuses::in_transit)",
+    "description": "Move is in transit with {{move.supplier.name}}"
```

Comment:
Seems a mix of tenses with the status description. This uses present tense whereas the rest seem to use past (has). I wonder if they should all use past, because in theory they have all occurred. Maybe `Move was started with {{supplier}}`?

Reply:
https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/998